apps/spider/crawlers/timetable.py

```diff
@@ -55,16 +54,16 @@ def crawl_timetable(term):
         preprocess=lambda x: re.sub("</tr>", "", x),
     )
     num_columns = len(soup.find(class_="data-table").find_all("th"))
-    assert num_columns == 20
+    assert num_columns == 21 # more than 20 after dartmouth added the lang requirement column
```

Comment:
Thanks for adjusting... parsing soup output and matching columns is no fun task.